test/unit/spec/common/cellComponents/tabs/jobStatusTable-Spec.js

```diff
@@ -106,6 +165,10 @@ define([
         if (!input) {
             input = job;
         }
+
```

Comment:
Added some extra criteria to the row testing to ensure that the row ID and the action button are set up as expected.